utils/protobuf-schema-utilities/src/main/java/io/apicurio/registry/utils/protobuf/schema/FileDescriptorUtils.java

```diff
@@ -400,6 +400,14 @@ private static MessageElement toMessage(FileDescriptorProto file, DescriptorProt
                     Collections.singletonList(reservedName));
             reserved.add(reservedElem);
         }
+        for (DescriptorProto.ReservedRange reservedRange : descriptor.getReservedRangeList()) {
+            List<Object> values = new ArrayList<>();
```

Comment:
Should this of `Integer` type instead of `Object`?

Reply:
Should be IntRange. I will change.